tests/nlp/test_text_data.py

```diff
@@ -141,11 +142,11 @@ def test_properties(text_classification_dataset_mock):
 
     # Act & Assert
     assert_that(dataset._properties, equal_to(None))  # pylint: disable=protected-access
-    dataset.calculate_default_properties(ignore_properties=['topic'])
+    dataset.calculate_default_properties(ignore_properties=['topic'] + HEAVY_PROPERTIES)
```

Comment:
So no test runs them now?
Maybe run them on a smaller test size? Just to make sure they work and that code is covered.
Also, test for the >10000 thingy

Reply:
Theres a pytorch conflict... `AssertionError: Unsupported PyTorch version, minimum required is 1.11.0, got: 1.10.2+cpu`

Reply:
https://github.com/deepchecks/deepchecks/issues/2415